Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Managers/VaultManager.swift

```diff
@@ -113,7 +171,7 @@ extension PrimerHeadlessUniversalCheckout {
             }
         }
         
-        public func startPaymentFlow(vaultedPaymentMethodId: String) {
+        public func startPaymentFlow(vaultedPaymentMethodId: String, vaultedPaymentMethodAdditionalData: PrimerVaultedPaymentMethodAdditionalData? = nil) {
```

Comment:
Would it be better to have two functions:

- `startPaymentFlow(vaultedPaymentMethodId: String)`
- `startPaymentFlow(vaultedPaymentMethodId: String, vaultedPaymentMethodAdditionalData: PrimerVaultedPaymentMethodAdditionalData)`

Reply:
The above is the same as having 2 functions, by providing a default input value the integrator can use exactly the 2 functions you posted above